config/webpack/loaders/erb.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  test: /\.erb$/,
```

Comment:
I didn't really want to change the default config in the same PR, since there are already some changes to those to just make it work.

But we will need to clean those defaults a bit, agreed there.. :)